spec/presenters/person_presenter_spec.rb

```diff
@@ -1,7 +1,8 @@
 # frozen_string_literal: true
 
 describe PersonPresenter do
-  let(:profile_user) { FactoryGirl.create(:user_with_aspect) }
+  let(:profile_user) { FactoryGirl.create(:user_with_aspect,
```

Comment:
Layout/MultilineBlockLayout: Block body expression is on the same line as the block start.